src/components/AddNewItem.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div>
+        <input v-model='itemToAdd' @keypress.enter="addItem()" placeholder="Add todo">
+        <button class="btn btn-primary" @click="addItem()">Add to todos!</button>    
+    </div>
+</template>
+
+<script>
+import uuid from 'uuid';
+import { mapActions, mapGetters } from  'vuex';
+import { Vue, Component} from 'vue-property-decorator'
+
+  @Component({
+    computed: {
+        ...mapGetters ([
+            'getItems'
+        ]),
+        itemToAdd: {
```

Comment:
In the components always use only the getters and the actions, the getters return the current state of the store, and the actions are designed to become a bridge between the component and the mutations. Only call the mutations inside the store, by an action.